tests/test_utils.py

```diff
@@ -0,0 +1,3 @@
+def check_item(asserted, assertion):
+    names = set(str(getattr(item, 'name', item)) for item in asserted)
```

Comment:
add `# TODO: After converting used_vars and used_attrs to sets, use dedicated tests for them.`